api/urls.py

```diff
@@ -1,12 +1,15 @@
+from django.conf.urls import include
 from django.urls import path
-from .views import TodoViewSet, LabelViewSet
+from .views import TodoViewSet, LabelViewSet, UserViewSet, UserCreateView
 from rest_framework import routers
 
-urlpatterns = []
+router = routers.DefaultRouter()
 
-router = routers.SimpleRouter()
 router.register('todos', TodoViewSet)
 router.register('labels', LabelViewSet)
+router.register('users', UserViewSet)
 
-# todo: user関連のrouter設定を一旦オミット。ユーザー登録、ログイン、ログアウトの実装に併せて修正か削除します。
-# router.register('users', UserViewSet)
+urlpatterns = [
+    path('', include(router.urls)),
+    path('sign_up/', UserCreateView.as_view()),
```

Comment:
```suggestion
    path('sign-up/', UserCreateView.as_view()),
```

URL にはアンダースコア(_)よりもハイフン(-)を使いましょう

https://support.google.com/webmasters/answer/76329?hl=ja

>URL では区切り記号を使うと効果的です。http://www.example.com/green-dress.html という URL の方が、http://www.example.com/greendress.html という URL よりずっとわかりやすくなります。URL にはアンダースコア（_）ではなくハイフン（-）を使用することをおすすめします。

Reply:
普通に知らなかった。以降留意します。